README.rst

```diff
@@ -34,6 +35,10 @@ This MFE is bundled with `Devstack <https://github.com/openedx/devstack>`_, see
 
    .. image:: ./docs/images/localhost_preview.png
 
+   Note: the alert will always display if subscriptions repo is not setup locally
+
+6. **(Optional)** For edx developers, they can set up the subscriptions repo for viewing subscription history
```

Comment:
Can you add a link to the subscriptions repo?

Reply:
Nervermind, the repo is a private repo so I don't think it's helpful for openedX.

Reply:
I avoided adding a link for similar reasons